Pension content design updates (proposed)

melwoodard opened the pull request:
Related: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2789

This PR proposes some small design changes for the WIP Pensions content. Changes include:

[Application Process](https://vetsgov-pr-5562.herokuapp.com/pension/apply/)

- Changed bolded text to headings

[Aid and Attendance](https://vetsgov-pr-5562.herokuapp.com/pension/eligibility/aid-attendance-housebound/)

- Changed bolded text to headings
- Removed extra gray container at bottom showing up w/o any piano key content

[Pension Management Center](https://vetsgov-pr-5562.herokuapp.com/pension/pension-management-center/)

- Made states list two-column for space efficiency on desktop
- Changed PMC names to headings

[Rates](https://vetsgov-pr-5562.herokuapp.com/pension/rates/)

- Changed text above rate tables to headings

[Survivors Pension](https://vetsgov-pr-5562.herokuapp.com/pension/survivors-pension/)

- Broke out content not intended to be in the blue callout
- Changed bolded text to headings

[Survivors Pension Rates](https://vetsgov-pr-5562.herokuapp.com/pension/survivors-pension/rates/)

- Changed text above rate tables to headings

Commit:
Headings and callout on /apply

Commit:
Headings and markdown simplification on A&A

Commit:
headings and multi-column lists on pmc page

Commit:
headings on pension rate page

Commit:
headings and adjustment to callout on survivors page

Commit:
headings on survivors pension rates

DanielleSoCompany commented:
In https://vetsgov-pr-5562.herokuapp.com/pension/pension-management-center/: It looks like in the last list (St. Paul), the line after the 2-column list is aligning to the right (under the 2nd column). Otherwise, changes look great to me!

melwoodard committed:
Fix layout bug and states list ordering

melwoodard commented:
@DanielleSoCompany I fixed the layout bug under St. Paul. I also noticed that while the lists read alphabetically left to right, they end up being out of order when they drop to one column on mobile. I changed the order so that it reads alphabetically down column 1 and continues down column 2, so that A-Z ordering is preserved no matter # of columns, like so:

Desktop:

<img width="530" src="https://cloud.githubusercontent.com/assets/4106356/26212124/6b221332-3ba9-11e7-99fc-537bddbe5a94.png">

Mobile:
_(I've noted the gap between the lists here, will get a fix in for it in the future)_

<img width="320" src="https://cloud.githubusercontent.com/assets/4106356/26212155/7e1bee68-3ba9-11e7-969d-85c3248ef59d.png">

DanielleSoCompany commented:
@melwoodard great point on the A-Z - thanks!
